include/test/battle.h

```diff
@@ -719,6 +719,7 @@ struct BattleTestRunnerState
     u8 checkProgressParameter;
     u8 checkProgressTrial;
     u8 checkProgressTurn;
+    u32 occurenceCounter;
```

Comment:
I'll have a think today about if there's something cleaner than adding a member to the test runner that's only used by one test.